src/modules/statusupdate.js

```diff
@@ -0,0 +1,21 @@
+import { dataStructure, render } from '../index.js'; // eslint-disable-line import/no-cycle
+
+export const newItem = document.getElementById('newItem');
```

Comment:
- [ ] Your code should have an event listener on the checkbox (change), and the task object's values for `completed` is updated upon user actions, kindly make sure this functionality is implemented and the new value is stored on the local storage, so the user can check an item and remain checked

Reply:
Hi, @Ismailco did you check the modules? Can you kindly check it cause I already have this functionality working on one of the modules file. I'll try to fix the cycle thing instead of ignoring it. Thanks